.github/workflows/foreman_plugin.yml

```diff
@@ -276,3 +276,4 @@ jobs:
         uses: re-actors/alls-green@release/v1
         with:
           jobs: ${{ toJSON(needs) }}
+          allowed-skips: existing-database
```

Comment:
Wonder if we should fold this one into the main "test-suite", but alas not today.